### PR TITLE
Small labels fixes

### PR DESCRIPTION
**Pull request by larslemos**

### CODE REVIEW

File: poc_config/openmrs/i18n/common/locale_pt.json [hunk 1]
```diff
@@ -177,9 +177,9 @@
     "COMMON_PATERNAL_A": "Pai A",
     "COMMON_PATERNAL_B": "Pai B",
     "COMMON_VITALS": "Vitais",
-    "COMMON_TEMPERATURE_C": "Temperatura (C)",
-    "COMMON_WEIGHT": "Peso",
-    "COMMON_HEIGHT": "Altura/Estatura",
+    "COMMON_TEMPERATURE_C": "Temperatura (ºC)",
+    "COMMON_WEIGHT": "Peso (Kg)",
+    "COMMON_HEIGHT": "Altura/Estatura (cm)",
```
Comment (valvijo) [inline poc_config/openmrs/i18n/common/locale_pt.json]: Height should be (m) meters

Comment (ynurmahomed) [reply poc_config/openmrs/i18n/common/locale_pt.json]: I think cm is right because we are using integers.

Comment (valvijo) [reply poc_config/openmrs/i18n/common/locale_pt.json]: Meters is the unit used by providers. I don't think we have space for changes here. If that is integer, then we should change to decimal

Comment (larslemos) [reply poc_config/openmrs/i18n/common/locale_pt.json]: @valvijo Can it be like this ?
image

Comment (valvijo) [reply poc_config/openmrs/i18n/common/locale_pt.json]: :+1::+1::+1::+1:

**Pull request merged**